src/Utils/RMC/RMST.as

```diff
@@ -434,8 +419,8 @@ class RMST : RMS {
 
         UI::BeginDisabled(RMC::ClickedOnSkip || isSwitchingMap);
         if (UI::Button(Icons::PlayCircleO + skipLabel)) {
-            RMC::IsPaused = true;
             RMC::ClickedOnSkip = true;
+            RMC::IsPaused = false;
```

Comment:
I think this should be "true".

Reply:
🤷‍♂️ This is what the other modes do (except, more verbosely: `if (RMC::IsPaused) RMC::IsPaused = false;`), so I included it here as a difference I noticed. Same goes for `RenderBrokenButton` further down.